src/custom-elements/src/components/phone-input/phone-input.tsx

```diff
@@ -170,6 +170,18 @@ export class ChiPhoneInput {
     this.chiChange.emit(this.value);
   };
 
+  _keyPressHandler = (event: KeyboardEvent): void => {
+    this._checkNoCharactersInInput(event);
```

Comment:
It still accepts some special chars - f.e. ¨^ó^¨ī¨^ī¨†¨ī¨ī^¨ó^Ļ^ó

https://user-images.githubusercontent.com/125541591/224738068-19cf64a1-1b92-4bc9-a0a1-d105961ef49f.mov